### PR TITLE
Sensitivity analysis batch

### LINKED ISSUE #152: SensitivityAnalysis: Run all batches in parallel.

### PR DESCRIPTION
**Pull request by PavelBal**
Fixes #152

Run all batches in parallel.

### CODE REVIEW

**Comment by PavelBal** [comment]
@IndrajeetPatil I have written the code to run all required simulations using as few simulation batches as possible, and running alll batches in parallel. You would have to re-write the following code to process the results.

All results have an ID, and mapping of parameter paths and their values is stored in a map. Let us have a quick chat about this.

**Review by IndrajeetPatil — APPROVED** [review]
@PavelBal Done. 

Everything should work as expected now, since the key data structure is still the same.

File: R/sensitivityCalculation.R [hunk 1]
```diff
@@ -82,48 +82,131 @@ sensitivityCalculation <- function(simulation,
...
+  # Create a nested list of simulation batch results.
+  # Outer list indexes each parameter path, while inner list corresponds to each
+  # value of parameter factor.
+  simulationResultsBatch <- batchResultsIdMap
+
+  for (parameterPath in seq_along(simulationResultsBatch)) {
+    for (parameterFactor in seq_along(simulationResultsBatch[[parameterPath]])) {
+      simulationResultsBatch[[parameterPath]][[parameterFactor]] <- purrr::pluck(simulationBatchesResults, batchResultsIdMap[[parameterPath]][[parameterFactor]])
+    }
+  }
```
Comment (IndrajeetPatil) [inline R/sensitivityCalculation.R:179]: The new `simulationResultsBatch` is the same as the old one, and so everything from here on works the same way as before.

Comment (PavelBal) [reply R/sensitivityCalculation.R:179]: Nice, that was easy!

**Pull request merged**